src/uwtools/tests/api/test_driver.py

```diff
@@ -1,25 +1,182 @@
-# pylint: disable=missing-function-docstring,protected-access
+# pylint: disable=missing-class-docstring,missing-function-docstring,protected-access,redefined-outer-name
 
-from inspect import isclass, ismodule
+import datetime as dt
+import logging
+from pathlib import Path
+from types import SimpleNamespace as ns
+from unittest.mock import patch
 
-from pytest import mark
+from pytest import fixture, mark, raises
 
 from uwtools.api import driver as driver_api
 from uwtools.drivers import driver as driver_lib
+from uwtools.exceptions import UWError
+from uwtools.logging import log
+from uwtools.tests.support import fixture_path, logged, regex_logged
+
+# Fixtures
+
+
+@fixture
+def args():
+    return ns(
+        classname="TestDriver",
+        config=fixture_path("testdriver.yaml"),
+        module="testdriver",
+        module_dir=fixture_path("testdriver.py").parent,
```

Comment:
Out of curiosity, why have you gone with two separate flags for the CLI (module name and module dir) instead of just having the user provide the module path and treat it like you do here?